jax/_src/numpy/lax_numpy.py

```diff
@@ -3479,6 +3479,27 @@ def polyadd(a1, a2):
     return a2.at[-a1.shape[0]:].add(a1)
 
 
+@_wraps(np.polyint)
+def polyint(p, m=1, k=None):
+  m = core.concrete_or_error(operator.index, m, "'m' argument of jnp.polyint")
+  p = asarray(p)
+  m = int(m)
+  if m < 0:
+    raise ValueError("Order of integral must be positive (see polyder)")
+  if k is None:
+    k = zeros(m, float)
+  k = atleast_1d(k)
```

Comment:
Should we check that k is rank-1? I think the code might do strange things if it's not.